error: ‘FILE’ does not name a type

--- origsrc/podofo-0.8.4/src/PdfInputDevice.h	2009-04-03 13:36:41.000000000 -0500
+++ src/podofo-0.8.4/src/PdfInputDevice.h	2010-11-01 13:07:00.668245900 -0500
@@ -21,6 +21,7 @@
 #ifndef _PDF_INPUT_DEVICE_H_
 #define _PDF_INPUT_DEVICE_H_
 
+#include <cstdio>
 #include <istream>
 
 #include "PdfDefines.h"
--- origsrc/podofo-0.8.4/src/PdfInputStream.h	2009-04-03 13:36:41.000000000 -0500
+++ src/podofo-0.8.4/src/PdfInputStream.h	2010-11-01 13:05:26.178881200 -0500
@@ -22,6 +22,7 @@
 #define _PDF_INPUT_STREAM_H_
 
 #include "PdfDefines.h"
+#include <cstdio>
 
 namespace PoDoFo {
 
--- origsrc/podofo-0.8.4/src/PdfOutputDevice.h	2009-05-02 06:37:17.000000000 -0500
+++ src/podofo-0.8.4/src/PdfOutputDevice.h	2010-11-01 13:06:10.155357800 -0500
@@ -22,6 +22,7 @@
 #define _PDF_OUTPUT_DEVICE_H_
 
 #include <cstdarg>
+#include <cstdio>
 #include <ostream>
 
 #include "PdfDefines.h"
--- origsrc/podofo-0.8.4/src/PdfOutputStream.h	2009-04-03 13:36:41.000000000 -0500
+++ src/podofo-0.8.4/src/PdfOutputStream.h	2010-11-01 13:05:33.666894200 -0500
@@ -24,6 +24,7 @@
 #include "PdfDefines.h"
 #include "PdfRefCountedBuffer.h"
 
+#include <cstdio>
 #include <string>
 
 namespace PoDoFo {
