--- origsrc/podofo-0.9.4/src/doc/PdfSigIncMemDocument.cpp	2016-05-15 12:18:26.000000000 -0500
+++ src/podofo-0.9.4/src/doc/PdfSigIncMemDocument.cpp	2016-08-07 22:30:35.595083700 -0500
@@ -35,6 +35,7 @@
 #pragma warning(disable: 4786)
 #endif
 
+#define _POSIX_C_SOURCE 200809L	/* strdup */
 #include <algorithm>
 #include <deque>
 #include <iostream>
--- origsrc/podofo-0.9.4/test/unit/TestUtils.cpp	2012-01-16 14:41:26.000000000 -0600
+++ src/podofo-0.9.4/test/unit/TestUtils.cpp	2016-08-07 22:34:06.858328400 -0500
@@ -17,6 +17,7 @@
  *   Free Software Foundation, Inc.,                                       *
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
+#define _POSIX_C_SOURCE 200112L	/* mkstemp */
 #include "TestUtils.h"
 
 #include <stdio.h>
