--- origsrc/podofo-0.9.5/CMakeLists.txt	2017-02-01 15:23:03.000000000 -0600
+++ src/podofo-0.9.5/CMakeLists.txt	2017-02-20 22:57:45.779523900 -0600
@@ -296,7 +296,7 @@ IF(CMAKE_COMPILER_IS_GNUCXX)
         SET(PODOFO_USE_VISIBILITY ${PODOFO_HAVE_GCC4})
     ENDIF(NOT DEFINED PODOFO_USE_VISIBILITY)
 
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++98")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++98")
 
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Woverloaded-virtual -Wswitch-enum -Wcast-qual -Wwrite-strings -Wredundant-decls -Wreorder")
 
